website/docs/cloud-docs/api-docs/policies.mdx

```diff
@@ -36,15 +36,15 @@ page_title: Policies - API Docs - Terraform Cloud
 
 # Policies API
 
```

Comment:
A thought --> if we can't explicitly comment out the OPA stuff here, then I think we need to add a comment somewhere visible about how OPA isn't yet available in TFE. Maybe we add on to that top note --> 

-> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud **Team & Governance** tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta. OPA Policies are also not available in Terraform Enterprise.